Tests/AwsSnsTests/AwsSnsTests.swift

```diff
@@ -83,16 +83,80 @@ class AwsSnsTests: XCTestCase {
     func testDeletePlatformEndpoint() {
         let deleteExpectation = expectation(description: "CreateExpectation")
         
-        let token = "225EF46104D58C43047A4B7749B41297A3A185CB9D441784AFEB5C2F1405285C"
+        let token = "225EF46104D58C43047A4B7749B41297A3185CB9D441784AFEB5C2F1405285C"
         
         snsClient?.createPlatformEndpoint(token: token, platformApplicationArn: "arn:aws:sns:us-west-2:487164526243:app/APNS_SANDBOX/Test") { success, endpointArn, error in
-            self.snsClient?.deleteEndpoint(endpointArn: endpointArn!) { success, error in
+            guard let endpointArn = endpointArn else {
+                XCTFail("CreatePlatformEndpoint returned nil.")
+                return
+            }
+            self.snsClient?.deleteEndpoint(endpointArn: endpointArn) { success, error in
                 XCTAssertNil(error, "DeletePlatformEndpoint returned error.")
                 deleteExpectation.fulfill()
             }
         }
         waitForExpectations(timeout: 5, handler: nil)
     }
+    
+    func testGetEndpointAttributes() {
+        let getAttributesExpectation = expectation(description: "GetAttributesExpectation")
+        
+        let token = "225EF46104D58C43047A4B7749B41297A3185CB9D441784AFEB5C2F1405285C"
+        
+        snsClient?.createPlatformEndpoint(token: token, platformApplicationArn: "arn:aws:sns:us-west-2:487164526243:app/APNS_SANDBOX/Test") { [weak self] success, endpointArn, _ in
+            guard success, let endpointArn = endpointArn else {
+                XCTFail("CreatePlatformEnpoint failed")
```

Comment:
we can always let the tests fail due to unfulfilled expectation, let's remove the code for failure in all methods 👍